boolector.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM hdlc/build:build AS build
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    binutils \
+    cmake \
+    autoconf \
+    git \
+ && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+ && update-ca-certificates  \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /tmp/boolector && cd /tmp/boolector \
+&& curl -fsSL https://codeload.github.com/boolector/boolector/tar.gz/master | tar xzf - --strip-components=1 \
```

Comment:
Please add one space at the beginning of L14-L24.

Reply:
Sure.

> Looks good!
> 
> Do you want to try adding it to the graph? See yices2: [dc677af](https://github.com/hdl/containers/commit/dc677af6914af9a77ae5891f14a595c07b0df9e5)

I will try.

Reply:
I tried to add `boolector` to the graph. Furthermore I fixed the `yices2` & `z3` `base` images to `:build`. Maybe you can have a look.